frontend/package.json

```diff
@@ -11,15 +11,17 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "14.1.0",
+    "@angular/cdk": "^14.1.0",
```

Comment:
What is this package for? always use specific versions do avoid using  ^or  ~

Reply:
Component Dev Kit package, added with angular material, dependency removed